packages/terra-doc-template/tests/wdio/doc-template-spec.js

```diff
@@ -1,39 +1,29 @@
-describe('DocTemplate', () => {
-  before(() => browser.setViewportSize(Terra.viewports('huge')[0]));
-
+Terra.describeViewports('DocTemplate', ['large'], () => {
```

Comment:
Why did this get updated to large when it previously tests huge?

Reply:
The PropsTable screenshot requires a custom taller viewport, which was being saved to `large`. It was also causing the screenshots taken after it to be `large` because they are no longer being passed a `viewports` option. I decided to make them all large.